private/sass/addons/_newsblog.scss

```diff
@@ -336,9 +336,6 @@
     .text-muted {
         color: $gray;
     }
-    a {
```

Comment:
copied over the styles from the current explorer theme, was already removed there.